src/foam/core/reflow/ReactiveDetailView.js

```diff
@@ -125,18 +130,32 @@ foam.CLASS({
       this.start().
         addClass(self.myClass('switch')).
         enableClass('reactive', self.reactive$).
-        on('click', self.toggleMode).
-        add(' = ').
-      end();
+        on('click', self.toggleMode)
```

Comment:
Seems odd that the previous line ends with . and the next line starts and this line has neither. Be consistent.